tree.scss

```diff
@@ -333,6 +324,7 @@ $timing-fn: ease-out;
       li.node:not(.transient):not(.selected):not(.transitioning-node) {
         .node-contents, .label-mask:after {
           opacity: .6;
+          @include transition(none);
```

Comment:
Yeah, added a comment in. That opacity causes the same weird transition bug in ie11. No idea why.